fs/fs.go

```diff
@@ -295,3 +325,83 @@ func getVfsStats(path string) (uint64, uint64, uint64, error) {
 	avail := uint64(s.Frsize) * s.Bavail
 	return total, free, avail, nil
 }
+
+func dockerStatusValue(status [][]string, target string) string {
+	for _, v := range status {
+		if len(v) == 2 && strings.ToLower(v[0]) == strings.ToLower(target) {
+			return v[1]
+		}
+	}
+	return ""
+}
+
+func dockerDMDevice(driverStatus string) (string, uint, uint, error) {
+	var config [][]string
+	err := json.Unmarshal([]byte(driverStatus), &config)
+	if err != nil {
+		return "", 0, 0, err
+	}
+	poolName := dockerStatusValue(config, "Pool Name")
+	if len(poolName) == 0 {
+		return "", 0, 0, fmt.Errorf("Could not get dm pool name")
+	}
+
+	poolName = path.Join("/dev/mapper", poolName)
+
+	dmTable, err := exec.Command("dmsetup", "table", poolName).Output()
```

Comment:
We can add it. 
I wonder if switching to alpine linux base image is necessary here. There is an [alpine linux package]( https://pkgs.alpinelinux.org/package/main/armhf/device-mapper) for this purpose